scripts/raceday_verify.py

```
@@ -0,0 +1,848 @@
+        "SELECT name FROM sqlite_master WHERE type='index' AND name=?", (name,)
+    ).fetchone()
+    return row is not None
+
+
+def run_fetch(spec, from_date, to_date, option, db):
+    cmd = [sys.executable, "-m", "src.cli.main", "fetch",
+           "--from", from_date, "--to", to_date,
+           "--spec", spec, "--option", str(option),
+           "--db", db, "--no-progress"]
+    print(f"  $ {' '.join(cmd)}")
+    r = subprocess.run(cmd, timeout=600)
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)